src/modules/ttlmonitor/test/ParserTest.cpp

```diff
@@ -106,8 +113,7 @@ TEST_F(TTLMonitorParserTest, ErrorHdrDataSize) {
 TEST_F(TTLMonitorParserTest, ErrorDataSize) {
   makeHeader(DataBadSize);
 
-  auto Res = parser.parse(PacketData);
-  ASSERT_EQ(Res, 0);
+  parser.parse(PacketData);
```

Comment:
Can the ASSERT_EQ(Res, 0) line be replaced with some error counter checks? Did res == 0 mean that no errors occurred while parsing?

Reply:
Yes, it really is redundant and was only used in tests. It returned the number of good readouts, but that is also covered in the stat counters and in the length ofthe array containing the list of successfully parsed readouts.

Reply:
Okay, as long as those counters are being checked than I'm happy with this change